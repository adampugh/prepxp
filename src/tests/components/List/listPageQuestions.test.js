import React from "react";
import { shallow } from "enzyme";

import ListPageQuestions from "../../../components/List/listPageQuestions";
import user from "../../fixtures/userLists";

describe("<ListPageQuestions />", () => {
    it("should match snapshot", () => {
        const wrapper = shallow(<ListPageQuestions questions={user.lists[0].questions} />);
        expect(wrapper).toMatchSnapshot();        
    });
});