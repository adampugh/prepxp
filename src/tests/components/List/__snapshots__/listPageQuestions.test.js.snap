// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListPageQuestions /> should match snapshot 1`] = `
<div
  className="listPage__questions"
>
  <div
    className="container-s"
  >
    <div
      className="question__item"
      key="What is the difference between let and var?0"
    >
      <h2>
        <span>
          Q
          1
        </span>
         
        What is the difference between let and var?
      </h2>
      <h2
        onClick={[Function]}
      >
        <i
          className="fas fa-trash-alt"
        />
      </h2>
    </div>
    <div
      className="question__item"
      key="Who created JavaScript?1"
    >
      <h2>
        <span>
          Q
          2
        </span>
         
        Who created JavaScript?
      </h2>
      <h2
        onClick={[Function]}
      >
        <i
          className="fas fa-trash-alt"
        />
      </h2>
    </div>
  </div>
</div>
`;
